src/test/java/com/wikia/webdriver/testcases/desktop/articlecrudtests/ArticleCRUDAnonTests.java

```diff
@@ -103,4 +106,14 @@ public void articleCRUDAnon_editArticleSecondTime() {
     visualEditMode.addContent(articleContent);
     visualEditMode.submitArticle();
   }
+
+  @Test(groups = {"ArticlePushNtimes"})
+  @Execute(onWikia = "my_wiki_name")
+  public void articlePushTest() {
+    int articlenum = 100;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "articlenum" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)